styles/style.css

```diff
@@ -0,0 +1,136 @@
+body {
+    background-color: #eceef1;
+}
+.top {
+    height: 70px;
+}
+.top-text {
+    float: left;
+    font-size: 0.9em;
+    line-height: 0.7;
+    text-align: right;
+    width: 90%;
+}
+
+.signin-btn {
+    display: inline-block;
+    margin-left: 8px;
+    margin-top: 8px;
+}
+
+.signin-btn button {
+    background-color: #0176c5;
+    border-radius: 3px;
+    color: white;
+    font-size: 1.5em;
+    padding: 5px;
+}
+
+header {
+    font-family: 'Lobster', cursive;
+    line-height: .7;
+    position: relative;
+    text-align: center;
+}
+
+h3 {
+    display: inline;
+}
+
+h3 a {
+    color: black;
+    text-decoration: none;
```

Comment:
I noticed you have `text-decoration: none` a few times maybe you can put this for all anchor tags and the one case where you want it (top right corner or webpage) then you'd add it using `text-decoration` property.

Reply:
This suggestion has been implemented.
Thanks.